src/Microsoft.DocAsCode.Common/Loggers/WarningCodes.cs

```diff
@@ -30,6 +30,8 @@ public static class Build
             public const string UidNotFound = "UidNotFound";
             public const string UnknownContentType = "UnknownContentType";
             public const string UnknownContentTypeForTemplate = "UnknownContentTypeForTemplate";
+            public const string IsFullBuild = "IsFullBuild";
+            public const string IsIncrementalBuild = "IsIncrementalBuild";
```

Comment:
put these into `InfoCodes.cs` as we are using `LogInfo`

Reply:
Changed